sklearn/preprocessing/_encoders.py

```diff
@@ -267,6 +327,13 @@ class OneHotEncoder(_BaseEncoder):
         .. versionchanged:: 0.23
            Added the possibility to contain `None` values.
 
+    infrequent_indices_ : list of arrays
```

Comment:
As a user, I might be primarily interested about the infrequent categories for introspection without doing any indexing wrangling. We can still keep `infrequent_indices_` to allow any potential wrangling required by user.

Reply:
Added a `infrequent_categories_` as a computed property to return the infrequent categories.